org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -115,3 +115,37 @@ GithubOidcSageBionetworksSecuityHubToJira:
     Account:
       - !Ref SecurityCentralAccount
     Region: us-east-1
+
+GithubOidcSageBionetworksAwsInfra:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: github-oidc-provider-access.njk
+  StackName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-awsinfra
+  Parameters:
+    GitHubOrg: "Sage-Bionetworks"
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    PolicyDocument: |
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+              "Sid": "ListObjectsInBucket",
+              "Effect": "Allow",
+              "Action": [ "s3:GetBucketLocation", "s3:ListBucket" ],
+              "Resource": [ "bootstrap-awss3cloudformationbucket-19qromfd235z9" ]
```

Comment:
Bucket was hardcoded in Travis, is there a way to get it from output of bootstrap in admincentral?

Reply:
unfortunately i can't think of a way to do that in this context.